components/FeatureHighlight/src/MDCFeatureHighlightViewController.h

```diff
@@ -150,7 +150,19 @@ typedef void (^MDCFeatureHighlightCompletion)(BOOL accepted);
  Enable legacy font scaling curves for Dynamic Type.
  Default value is NO.
  */
-@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:) BOOL mdc_legacyFontScaling;
+@property(nonatomic, readwrite, setter=mdc_setLegacyFontScaling:)
+    BOOL mdc_legacyFontScaling __deprecated;
```

Comment:
Should this pass-through to the new API? That way they both have the same values? If not, it seems like you have to add temporary code to check their states and the interaction seems non-trivial.

If `mdc_legacyFontScaling` is `YES` but `adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable` is `NO`, should you use the fall-back scaling or not?